charts/ia-home-office-integration-api/values.yaml

```diff
@@ -5,6 +5,9 @@ java:
   environment:
     IDAM_URL: "https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net"
     S2S_URL: "http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    HOME_OFFICE_ENDPOINT: "http://ia-home-office-mock-api-aat.service.core-compute-aat.internal"
```

Comment:
you may use `{{ .Values.global.environment }}` instead of hardcoded `aat`

Reply:
It fails on PR if it connects to preview url

Reply:
This will be replaced later by the home office api url

Reply:
I'm not sure this is resolved?  Should work since the other URLs above do.

Reply:
This is changed in PR #77